include/fastq_conversion.hpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+
+void convert_fastq(std::filesystem::path fastq_file, std::filesystem::path out)
+{
+    seqan3::sequence_file_input fin{fastq_file};
+    seqan3::sequence_file_output fout{std::cout, seqan3::format_fasta{}};
+    if (out != "") // If an output path is given
+        seqan3::sequence_file_output fout{out/"output.fasta"};
+
+    fout = fin; // conversion
+}
```

Comment:
Please write function definitions in a corresponding cpp file, because this is an app and not a library. You can look into the give_me_five files for an example. It would also be great to have the functions in a namespace.

Reply:
I thought, we don't want to use namespaces anymore?

Reply:
Yes, you're right. Such a small app does not need a namespace.